source/codegen/metadata/nirfmxbluetooth/attributes.py

```diff
@@ -198,6 +198,11 @@
         'name': 'VHDT_MODE_ENABLED',
         'type': 'int32'
     },
+    11534401: {
```

Comment:
For all of these, can you also update the config.py to reflect the updated version we're generating from.